README.md

```diff
@@ -36,95 +35,46 @@ This project builds upon work done by [Shobhitaa](https://github.com/shobhitaa),
 We're aiming to make the JoinMarket Web UI available for different node systems.
 If your node of choice is missing, feel free to integrate it and let us know so we can add it here.
 
-- [**RaspiBlitz**](https://github.com/rootzoll/raspiblitz): A beta version of the JoinMarket Web UI will be [available in RaspiBlitz v1.7.2](https://github.com/rootzoll/raspiblitz/pull/2747). We're aiming for a production ready version to be available with [RaspiBlitz v1.8.0](https://github.com/rootzoll/raspiblitz/pull/2747#issuecomment-1013866678).
-- [**Citadel**](https://runcitadel.space/): Citadel integration is work in progress.
-- [**Umbrel**](https://getumbrel.com/): Umbrel integration is [work in progress](https://github.com/getumbrel/umbrel/pull/1216).
+### [RaspiBlitz](https://github.com/rootzoll/raspiblitz)
 
-## ✨ Features
-
-- [x] Spending from the wallet without Coinjoin
-- [x] Simple CoinJoins
-- [x] Running the yield generator
-- [ ] TBD
-
-## 👩‍🔧 Installation
-
-These instructions assume you want to run the web UI locally and connect it to a JoinMarket instance on your RaspiBlitz.
-See [DEVELOPING.md](DEVELOPING.md)</a> for setting up a regtest development environment.
-
-### 🚨 Prerequisite: JoinMarket on RaspiBlitz
-
-To run the web UI locally you need to connect it to a running JoinMarket instance.
-
-#### 1. JoininBox
-
-Install [JoininBox](https://github.com/openoms/joininbox) on your [RaspiBlitz](https://github.com/rootzoll/raspiblitz):
+The alpha version of the JoinMarket Web UI is [available as CLI install in RaspiBlitz v1.7.2](https://github.com/rootzoll/raspiblitz/pull/2747). To install it, exit the Raspiblitz menu and run:
 
 ```
-Services > j [BTC JoinMarket+JoininBox menu]
+config.scripts/bonus.joinmarket-webui.sh on
```

Comment:
Due to #121, we'll probably have to change this to:

```suggestion
config.scripts/bonus.joinmarket-webui.sh on
```

followed by:

```
```
config.scripts/bonus.joinmarket-webui.sh update
```
```